qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -181,7 +183,7 @@ def __init__(
         self._quantum_instance = None
         if quantum_instance is not None:
             with warnings.catch_warnings():
-                warnings.simplefilter("ignore", category=PendingDeprecationWarning)
+                warnings.simplefilter("ignore")
```

Comment:
I think it will be rendered because we indeed are calling the setter. I don't see much downside to these 2 lines, but there's a big upside in being confident they won't see a "double warning".  The `@deprecate_arg` here is all we want them to see.

But I'll add back the `category` argument.